vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md

```diff
@@ -107,55 +116,63 @@ ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1
 ```
 
 ### Verify the NGINX Ingress Install
-We recommend setting up a test application so you can easily verify the ingress is working in your cluster. See [this page](../google-cloud-platform/gke-install#verify-the-nginx-ingress-install) for those steps. You can use your local `kubectl`.
+We recommend verifying nginx is working correctly. The simplest option is to access the EXTERNAL-IP in your browser. You should get a `404 Not Found` nginx error page. Alternatively you can set up a simple test application. See [this page](../google-cloud-platform/gke-install#verify-the-nginx-ingress-install) for those steps. You can use your local `kubectl` for that work.
 
 ### Install the Entando Custom Resource Definitions (CRDs)
 Once per cluster you need to deploy the `Entando Custom Resources`.
 1. Download the Custom Resource Definitions (CRDs) and deploy the cluster scoped resources
-```
+```sh
 kubectl apply -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
 ```
 Next you can create a namespace for the Entando Application.
 
 2. Create the namespace
-```
+```sh
 kubectl create namespace entando
 ```
 3. Install the namespace scoped resources
-```
+```sh
 kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml
 ```
+If you run `kubectl -n entando get pods` you'll see the Entando operator is now running.
 
```

Comment:
think there should be a comma after 'pods'.

Reply:
Yep, done